package.json

```diff
@@ -3,23 +3,16 @@
   "version": "3.0.0",
   "description": "Automatically adds the target and rel attributes to external links",
   "license": "MIT",
-  "keywords": [
-    "markdown",
-    "remark",
-    "plugin",
-    "link",
-    "url"
-  ],
+  "keywords": ["markdown", "remark", "plugin", "link", "url"],
```

Comment:
Could you drop these unnecessary style changes, except for adding your name?

Reply:
Ugh Prettier!